geniesp/bpc_redcap_export_mapping.py

```diff
@@ -844,7 +844,7 @@ def run(self):
         # Must use this to determine release scope!
         data_elements = self.syn.tableQuery(
             f"select distinct variable from {self._DATA_ELEMENT_SYN_ID} "
-            f'where "{self._SPONSORED_PROJECT}-sor" in '
+            f'where "{self._SPONSORED_PROJECT}_sor" in '
```

Comment:
Why the change now?

Reply:
@xindiguo updated in source table.   I think the hyphen was causing issues with table queries with other scripts for some reason.